testutils/gke/fake/client.go

```diff
@@ -172,6 +173,19 @@ func (fgsc *GKESDKClient) GetCluster(project, region, zone, cluster string) (*co
 	return nil, fmt.Errorf("cluster not found")
 }
 
+// ListClustersInProject lists all the GKE clusters created in the given project.
+func (fgsc *GKESDKClient) ListClustersInProject(project string) ([]*container.Cluster, error) {
+	allClusters := make([]*container.Cluster, 0)
```

Comment:
you can do `make(x, 0, len(fgsc.clusters))`

Reply:
Hmm, we can not predetermine its length, because in the following for-loop, we need to append all elements in the slices to it.